bluebottle/bb_tasks/permissions.py

```diff
@@ -21,40 +21,48 @@ def get_parent_from_request(self, request):
 
         return parent
 
-    def has_object_method_permission(self, method, user, view, obj):
+    def has_object_action_permission(self, action, user, obj):
         return user == obj.owner
 
 
-class MemberOrTaskOwnerOrReadOnlyPermission(BasePermission):
-    # TODO: Move this check to the serialiser
-    def _time_spent_updated(self, request, task_member):
-        if request.data:
+class TaskMemberTimeSpentPermission(BasePermission):
```

Comment:
@eodolphi this permissions class won't work with the PermissionField because the logic is located in the has_object_permission and not in has_object_action_permission. This is like a field level permission - something to think about later.